utils/outputwriter/outputwriter.go

```diff
@@ -123,10 +123,12 @@ type descriptionBullet struct {
 	value string
 }
 
-func createVulnerabilityDescription(vulnerability *formats.VulnerabilityOrViolationRow) string {
-	var cves []string
-	for _, cve := range vulnerability.Cves {
-		cves = append(cves, cve.Id)
+func createVulnerabilityDescription(vulnerability *formats.VulnerabilityOrViolationRow, cves []string) string {
+	descriptionBullets := []descriptionBullet{
+		{title: "**Severity**", value: fmt.Sprintf("%s %s", xrayutils.GetSeverity(vulnerability.Severity, xrayutils.ApplicableStringValue).Emoji(), vulnerability.Severity)},
```

Comment:
**Severity**  >> 
markdownBold("Sevirity")

and maybe there is a markdown library already for go?